xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -224,6 +245,88 @@ protected WebSocketClientHandler getWebSocketClientHandler(
 
   private WebSocketClientHandler.WebSocketMessageHandler channelInactiveHandler = null;
 
+  String submitLimitOrder(KrakenStreamingLimitOrder krakenStreamingLimitOrder) {
```

Comment:
also, I see what you have effectively done here is make a non-blocking operation blocking, would it make more sense to have this method return the observable and let the caller deal with the async behavior, it may not, but if a user wants a blocking operation I would imagine they could just use REST?

Reply:
Well, the difference between sending the command via REST and via Websocket is that.... it is sent via Websocket. Just different channel, much faster though usually. The rest is exactly the same as in REST. Command sent, I want the response back I don't do anything until I get the response for my command therefore this is blocking.